tools/image_processing/imagej2/imagej2_bunwarpj_compare_elastic/imagej2_bunwarpj_compare_elastic.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_bunwarpj_align" name="Compare two elastic deformations" version="1.0.0">
+    <description>by warping index with bUnwarpJ</description>
+    <macros>
+        <import>imagej2_macros.xml</import>
+    </macros>
+    <expand macro="fiji_headless_requirements" />
+    <command>
+<![CDATA[
+    python $__tool_directory__/imagej2_bunwarpj_compare_elastic.py
+    --target_image "$target_image"
+    --target_image_format $target_image.ext
+    --source_image "$source_image"
+    --source_image_format $source_image.ext
+    --target_transformation "$target_transformation"
+    --source_transformation "$source_transformation"
+    --max_heap_size_type $set_max_heap_size.max_heap_size_type
+    --max_heap_size $set_max_heap_size.max_heap_size
+    --output "$output"
+]]>
+    </command>
+    <inputs>
+        <param format="bmp,eps,gif,jpg,pcx,pgm,png,psd,tiff" name="target_image" type="data" label="Target image"/>
+        <param format="bmp,eps,gif,jpg,pcx,pgm,png,psd,tiff" name="source_image" type="data" label="Source image"/>
+        <param format="txt" name="target_transformation" type="data" label="Target inverse transformation"/>
+        <param format="txt" name="source_transformation" type="data" label="Source direct transformation"/>
+        <expand macro="max_heap_size_type_conditional" />
+    </inputs>
+    <outputs>
+        <data name="output" format="txt" label="${tool.name} on ${on_string}: Warping index" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="target_image" value="dotblot.jpg" />
+            <param name="source_image" value="blobs.gif" />
+            <param name="target_transformation" value="target_transf1.txt" />
+            <param name="source_transformation" value="source_transf1.txt" />
+            <output name="output" file="warping_index.txt" />
+        </test>
+    </tests>
+    <help>
+**What it does**
+
+<![CDATA[
+Calculates the warping index of two elastic transformations, i.e. the average of the geometrical distance
+between every pixel and its version after applying both transformations (direct and inverse). This value
+is given as a result. The transformation files must be in the same format format as the files created with
+the **Save Transformation** option in the **Align two images with bUnwarpJ** tool.
+
+]]>
+
+    </help>
+    <citations>
+        <citation type="bibtex">
+            @InProceedings(Arganda-Carreras2006,
+                author =     "Ignacio Arganda-Carreras and
+                                    Carlos Oscar S{\'a}nchez Sorzano and
```

Comment:
@bgruening Yes, it could, although this only applies to the bunwarpj tools.  There are enough of them that this should (and will) go to the macro file.  I'll make that change for all bunwarpj tools in my next commit if that''s ok.